pkg/notes/adaptor/repository/dummy.ts

```diff
@@ -166,19 +166,41 @@ export class InMemoryNoteAttachmentRepository
   }
 }
 
+type NoteID_AccountID = string;
```

Comment:
```suggestion
type CompositeKey = `${NoteID}_${AccountID}`;
```
こういうふうにテンプレートリテラル型として書いたほうがよさそうです.